images/adapter/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian:stable
+
+RUN apt-get -q -y update && \
+    apt-get install --no-install-recommends -y -q \
+    procps \
+    curl \
```

Comment:
Same as above - I don't think you need all this stuff inside the container.

Reply:
will do

Reply:
applied